libs/src/sanityChecks/rolloverNodes.js

```diff
@@ -1,69 +1,88 @@
+const _ = require('lodash')
+
 const Utils = require('../utils')
 const CreatorNode = require('../services/creatorNode')
 
 const THREE_SECONDS = 3000
 const MAX_TRIES = 3
 
-/** Check if the user's primary creator node is healthy */
-const checkPrimaryHealthy = async (libs, primary, tries) => {
-  const healthy = await Utils.isHealthy(primary)
+const nodeIsHealthy = async (endpoint, tries) => {
```

Comment:
all changes for this function are just naming. we use this function to check all nodes not just primary, so changed name to reflect that.